apps.json

```diff
@@ -3004,6 +3004,17 @@
         ],
         "url": "https://github.com/YunoHost-Apps/yellowcms_ynh"
     },
+    "yeswiki": {
+        "branch": "master",
+        "category": "publishing",
+        "revision": "HEAD",
```

Comment:
Please if you set `HEAD` create and work on the branch `testing`

Reply:
Sorry, it's ok I create branch `testing`